logger/mocklogger_test.go

```diff
@@ -0,0 +1,67 @@
+package logger
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	router "gopkg.in/Clever/kayvee-go.v5/router"
+)
+
+func TestMockLoggerImplementsKayveeLogger(t *testing.T) {
+	assert.Implements(t, (*KayveeLogger)(nil), &MockRouteCountLogger{}, "*MockRouteCountLogger should implement KayveeLogger")
+}
+
+func TestRouteCountsWithMockLogger(t *testing.T) {
+	routes := map[string](router.Rule){
+		"rule-one": router.Rule{
+			Matchers: router.RuleMatchers{
+				"foo": []string{"bar", "baz"},
+			},
+			Output: router.RuleOutput{
+				"out": "#-%{foo}-",
+			},
+		},
+		"rule-two": router.Rule{
+			Matchers: router.RuleMatchers{
+				"abc": []string{"def"},
+			},
+			Output: router.RuleOutput{
+				"more": "x",
+			},
+		},
+	}
+	testRouter := router.NewFromRoutes(routes)
+
+	mockLogger := NewMockCountLogger("testing")
```

Comment:
my intention was to recommend `NewMockCountLogger()` so that people don't have to understand formatter / output details just to mock. that said, either way is doable if formatLogger is exported. 

👍 to README example